presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1005,24 +1005,38 @@ public void testTime(boolean insertWithPresto, ZoneId sessionZone)
     @Test(dataProvider = "testTimestampDataProvider")
     public void testTimestamp(boolean insertWithPresto, ZoneId sessionZone)
     {
-        // using two non-JVM zones so that we don't need to worry what Postgres system zone is
-        DataTypeTest tests = DataTypeTest.create(true)
-                .addRoundTrip(timestampDataType(), beforeEpoch)
-                .addRoundTrip(timestampDataType(), afterEpoch)
-                .addRoundTrip(timestampDataType(), timeDoubledInJvmZone)
-                .addRoundTrip(timestampDataType(), timeDoubledInVilnius)
-                .addRoundTrip(timestampDataType(), epoch) // epoch also is a gap in JVM zone
-                .addRoundTrip(timestampDataType(), timeGapInJvmZone1)
-                .addRoundTrip(timestampDataType(), timeGapInJvmZone2)
-                .addRoundTrip(timestampDataType(), timeGapInVilnius)
-                .addRoundTrip(timestampDataType(), timeGapInKathmandu);
+        DataTypeTest tests = DataTypeTest.create(true);
+        for (int precision : List.of(3, 6)) {
+            // test all standard cases with precision 3 and 6 to make sure the long and short TIMESTAMP WITH TIME ZONE
```

Comment:
"TIMESTAMP WITH TIME ZONE"

also, is 3 and 6 actually different cases?
i think in timestamp case, we go with the same code path

(ok to leave the test coverage, but if true, change the commnt)

Reply:
Oh - you noticed it too - yeah - it same case. I dropped the coverage.